openhtf/output/web_gui/__init__.py

```diff
@@ -445,6 +445,16 @@ def initialize(self, port):  # pylint: disable=arguments-differ
     def get(self):
       self.render('index.html', host=socket.gethostname(), port=self.port)
 
+  class ConfigHandler(tornado.web.RequestHandler):
+    """Provides configuration information.
+
+    This exposes the config file or flags used to run the web GUI.
+    Note that this is distinct from the configuration of test stations.
+    """
+
+    def get(self):
+      self.write(conf._asdict())
```

Comment:
Can you make this a dict so we can potentially add non-conf variables to it and stay wire compatible?